projects/emissary-ingress/emissary/helm/patches/0001-helm-changes.patch

```diff
@@ -4501,15 +4717,327 @@ index 2d06c8127..f1e043136 100644
            imagePullPolicy: {{ .Values.image.pullPolicy }}
            ports:
              {{- range .Values.service.ports }}
+@@ -190,7 +190,7 @@ spec:
+             {{- end }}
+             - name: AMBASSADOR_NAMESPACE
+               {{- if .Values.namespace.name }}
+-              value: {{ .Values.namespace.name | quote }}
++              value: {{ .Values.namespace.name |.Release.Namespace | default .Values.defaultNamespace | quote }}
+               {{ else }}
+               valueFrom:
+                 fieldRef:
+@@ -200,8 +200,8 @@ spec:
+               value: {{ include "ambassador.fullname" . }}-agent-cloud-token
+             {{- if .Values.env }}
+             {{- range $key,$value := .Values.env }}
+-            - name: {{ $key | upper | quote}}
+-              value: {{ $value | quote}}
++            - name: {{ $key | upper | .Release.Namespace | default .Values.defaultNamespace | quote}}
```

Comment:
This is incorrect, it removes env variables.

Reply:
good catch